litmus-portal/graphql-server/graph/schema.graphqls

```diff
@@ -447,5 +449,5 @@ type Subscription {
   #It is used to listen cluster operation request from the graphql server
   clusterConnect(clusterInfo: ClusterIdentity!): ClusterAction!
 
-  getKubeObject(kubeObjectRequest: KubeObjectRequest!): KubeObjectResponse! @authorized
```

Comment:
We don't need authorization for this query?

Reply:
I missed that, adding it!. Thanks @Jonsy13